test/testers/rust-tester/src/pdre_api/mod.rs

```diff
@@ -0,0 +1,46 @@
+mod crypto;
+mod network;
+mod storage;
+mod utils;
+
+pub fn run_full_tests() {
+    println!("***** PDRE API Test Suite *****");
+    println!("NOTE: not all tests display information, only those deemed necessary. If a test fails, this program panics.");
+    println!("");
+
+    // crypto tests
+    println!("Running crypto tests...");
```

Comment:
So here I think something like this is expected https://github.com/w3f/polkadot-spec/blob/6d9d4b542e7a8bd8ab55fd2d95db1daf5745b928/test/testers/rust-tester/src/trie_tester.rs#L144 which decides which test to be run depending from the input given by the command line.

Reply:
Done!